calculator.js

```diff
@@ -1,5 +1,140 @@
-const exampleAdditionInput = {
+// Set array of valid operations 
+const validOperations = ['+', 'add', '-', 'subtract', '/', 'divide', '*', 'multiply']
+
+// Helper functions
+function validateNums(input) {
+  if (isNaN(input.num1) || isNaN(input.num2)) {
+    return false; 
+  } else {
+    return true; }
+}
+
+function validateOperator(input) {
+  if (!validOperations.includes(input.operation)) {
+    return false; }
+  else {
+    return true; }
```

Comment:
The closing curly brace of an if statement should go on the next line. So the syntax for an if/else would be:

```js
if (condition) {
  statement;
} else {
  statement;
}
```

It looks like you've got this lined up correctly elsewhere in this file, it's just in these validation functions that it's a problem.